Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -1555,7 +1555,22 @@ def panorama_create_address_command(args: dict):
     """
     address_name = args['name']
     description = args.get('description')
-    tags = argToList(args['tag']) if 'tag' in args else None
+    if tags := argToList(args.get('tag', [])):
+        xpath = "/config/devices/entry[@name='localhost.localdomain']"
+        if DEVICE_GROUP:
+            xpath += f"/device-group/entry[@name='{DEVICE_GROUP}']/tag"
+        if VSYS:
+            xpath += "/vsys/entry[@name='vsys1']/tag"
+        result = http_request(URL, 'GET', params={'type': 'config', 'action': 'get', 'key': API_KEY, 'xpath': xpath})
```

Comment:
please extract this into a function so it can be reused.

Reply:
I found the `XPATH_OBJECTS` global variable which makes the job.

Reply:
I found the `XPATH_OBJECTS` global variable which makes the job.